Parsers/ASimAuditEvent/Parsers/ASimAuditEvent.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
```

Comment:
why there are KQL tests related changes?

Reply:
i'm updating the schema table so kql tests is changing

Reply:
which schema table alekhya?